lib/models/patient_source.js

```diff
@@ -0,0 +1,15 @@
+module.exports = class PatientSource {
```

Comment:
Add a comment similar to the version in Bonnie, saying that these functions are required by the cql-execution engine.